src/main/scala/service/IssuesService.scala

```diff
@@ -139,11 +210,16 @@ trait IssuesService {
         .map { case (owner, repository) => t1.byRepository(owner, repository) }
         .foldLeft[Column[Boolean]](false) ( _ || _ ) &&
       (t1.closed           === (condition.state == "closed").bind) &&
-      (t1.milestoneId      === condition.milestoneId.get.get.bind, condition.milestoneId.flatten.isDefined) &&
-      (t1.milestoneId.?    isEmpty, condition.milestoneId == Some(None)) &&
+      //(t1.milestoneId      === condition.milestoneId.get.get.bind, condition.milestoneId.flatten.isDefined) &&
```

Comment:
umm.. this change contain https://github.com/team-lab/gitbucket/commit/e67365a19f58b0034c23e02fe48463aaef30ac9a by @takezoe ...